presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUpgrader.java

```diff
@@ -81,9 +84,10 @@ private static void upgradeUsingRpm(String pathToRpm, boolean disableDependencyC
         if (disableDependencyChecking) {
             nodeps = "--nodeps";
         }
-        String upgradeRpm = format("sudo rpm -U %s %s", nodeps, pathToRpm);
+        String upgradeRpm = format("sudo rpm -Uv %s %s", nodeps, pathToRpm);
```

Comment:
Having -v flag provides useful information like
Package already installed. 
It provides two lines of useful information for debugging.